### PR TITLE
Initial checking for Customer Sales connector

### PR DESCRIPTION
**Pull request by amancdk**
---
Please check the following conditions for your PR.

- [ ] `apiDefinition.swagger.json` is validated using `paconn validate` command.
- [ ] `apiProperties.json` has a valid brand color. Invalid brand colors are `#007ee5` and `#ffffff`.

### CODE REVIEW

File: custom-connectors/EleadCustomerSales/apiProperties.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#1F1A3C",
+    "capabilities": [],
+    "publisher": "Bhargava, Aman Avinash"
```
Comment (dougelliott) [inline custom-connectors/EleadCustomerSales/apiProperties.json:6]: The `stackOwner` property is missing. Additionally, for a certified connector, these fields should not contain PII, but instead point to the wider company the publisher and stackowner is.

File: custom-connectors/EleadCustomerSales/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,1909 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Elead Sales Customers",
+    "version": "1.0",
+    "description": "This API allows for the management of prospect/customer information inside the Elead CRM.This API includes the following capabilities and features for managing prospect/customer information, Add prospect/customer information such as name, phone numbers, email addresses and physical address and return the uniquely-assigned Elead ID on post, Update, search and retrieve prospect/customer contact information tied to an existing Elead ID.",
```
Comment (dougelliott) [inline custom-connectors/EleadCustomerSales/apiDefinition.swagger.json:6]: There is a missing space after the first sentence's period and the second sentence of the description, between `CRM.` and `This`

**Review by dougelliott — CHANGES_REQUESTED** [review]
In addition to the comments, please put these files in the `certified-connectors` directory, not the `custom-connectors` directory

**Comment by sriyen-msft** [comment]
@amancdk Please remove the icon and move the connector folder to `certified-connectors` folder

**Review by sriyen-msft — CHANGES_REQUESTED** [review]
@amancdk Please remove the icon and move the connector folder to certified-connectors folder

**Comment by amancdk** [comment]
@sriyen-msft & @dougelliott  made suggested changes

**Review by dougelliott — APPROVED** [review]

**Pull request merged**